src/TwilioConfig.php

```diff
@@ -4,92 +4,78 @@
 
 class TwilioConfig
```

Comment:
I would rather drop this entire class. It is a mere wrapper around an array, which serves for reading only.
It is better to use `Arr` helper at `Twilio` directly.

Reply:
I think it's still useful, as it provides type safety and a place to put logic such as checking if a code is ignored/etc.